### DISCUSSION TITLE
Need help with using DBeaver PostgreSQL

### DISCUSSION THREAD

**Discussion by davidmory38** [dbeaver/dbeaver#11050] [General]
For some reason (using the latest version of 7.2.0 DBeaver) I can't for the life of me find the option to create a new table/view. I've googled around and found guides but wherever the guide shows right clicking giving you the "create new table" option, I don't have it on my

New to all of this stuff so getting stuck on such a basic/simple first step is beyond frustrating. Someone help please!

**Comment by uslss** [comment]
Probably you switch to the Simple connection view mode. Just try to change it to advanced using connection context menu in Database Navigator
![image](https://user-images.githubusercontent.com/31996417/106107067-810b9100-6157-11eb-9a57-8a2afd61a71e.png)
